textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/join/JoinTest.java

```diff
@@ -1,2015 +0,0 @@
-        if (outerTuple == null) {
-            ;
-        } else {
-            dataWriterForOuter.open();
-            for (ITuple tuple : outerTuple) {
-                dataWriterForOuter.insertTuple(tuple);
-            }
-            dataWriterForOuter.close();
-        }
-        if (innerTuple == null) {
-            return;
-        }
-        dataWriterForInner.open();
-        for (ITuple tuple : innerTuple) {
-            dataWriterForInner.insertTuple(tuple);
-        }
-        dataWriterForInner.close();
-    }
-
-    // A helper method to setup the test cases.
-    // Types allowed (as of now) are:
-    // index -> CONJUNCTION_INDEXBASED KeywordMatcher
-    // phrase -> PHRASE_INDEXBASED KeywordMatcher
-    //
-    // whichOperator is to specify either "outer" or "inner" operator.
-    public IOperator setupOperators(String query, String type, String whichOperator) throws DataFlowException {
-        KeywordPredicate keywordPredicate = null;
-        IDataStore dataStore = null;
-        switch (type) {
-        case "index":
-            if (whichOperator == "outer") {
-                dataStore = dataStoreForOuter;
-                keywordPredicate = new KeywordPredicate(query, 
-                        Utils.getAttributeNames(modifiedAttributeList), analyzer,
-                        DataConstants.KeywordMatchingType.CONJUNCTION_INDEXBASED);
-            } else if (whichOperator == "inner") {
-                dataStore = dataStoreForInner;
-                keywordPredicate = new KeywordPredicate(query, 
-                        Utils.getAttributeNames(modifiedAttributeList), analyzer,
-                        DataConstants.KeywordMatchingType.CONJUNCTION_INDEXBASED);
-            }
-            break;
-        case "phrase":
-            if (whichOperator == "outer") {
-                dataStore = dataStoreForOuter;
-                keywordPredicate = new KeywordPredicate(query, 
-                        Utils.getAttributeNames(modifiedAttributeList), analyzer,
-                        DataConstants.KeywordMatchingType.PHRASE_INDEXBASED);
-            } else if (whichOperator == "inner") {
-                dataStore = dataStoreForInner;
-                keywordPredicate = new KeywordPredicate(query, 
-                        Utils.getAttributeNames(modifiedAttributeList), analyzer,
-                        DataConstants.KeywordMatchingType.PHRASE_INDEXBASED);
-            }
-            break;
-
-        default:
-            break;
-        }
-
-        KeywordMatcherSourceOperator keywordSource = new KeywordMatcherSourceOperator(keywordPredicate, dataStore);
-
-        return keywordSource;
-    }
-
-    // A helper method to populate tuples' list to query upon. Currently
-    // consists of two sets/lists of tuples with five tuples in each.
-    // Takes in the set number and number of tuples.
-    public List<ITuple> setupTuplesList(int whichList, int numberOfTuples) {
```

Comment:
The old function includes some test data. These data are moved to `JoinTestConstants.java`.